src/ert/_c_wrappers/enkf/ert_run_context.py

```diff
@@ -12,42 +14,65 @@
 @dataclass
 class RunContext:
     sim_fs: "EnkfFs"
-    target_fs: "EnkfFs" = None
-    mask: List[bool] = field(default_factory=list)
-    paths: List[str] = field(default_factory=list)
-    jobnames: List[str] = field(default_factory=list)
+    path_format: str
+    format_string: str
+    runpath_file: Path
+    initial_mask: List[bool] = field(default_factory=list)
+    global_substitutions: Optional[Dict[str, str]] = None
+    job_name: str = ""
```

Comment:
Where is this used? Perhaps a mistake? Don't see a reason `RunContext` should have a `job_name` property?